cs/client.py

```diff
@@ -234,20 +248,16 @@ def read_config(ini_group=None):
     # Look at CLOUDSTACK_CONFIG first if present
     if 'CLOUDSTACK_CONFIG' in os.environ:
         paths += (os.path.expanduser(os.environ['CLOUDSTACK_CONFIG']),)
-    if not any([os.path.exists(c) for c in paths]):
-        raise SystemExit("Config file not found. Tried {0}".format(
-            ", ".join(paths)))
+
     conf = ConfigParser()
     conf.read(paths)
-    try:
-        cs_conf = conf[ini_group]
-    except AttributeError:  # python 2
-        cs_conf = dict(conf.items(ini_group))
-    cs_conf['name'] = ini_group
 
-    allowed_keys = ('endpoint', 'key', 'secret', 'timeout', 'method', 'verify',
-                    'cert', 'name', 'retry', 'theme')
+    if not conf.has_section(ini_group):
+        return dict()
```

Comment:
mmh why not a simple `return` (`None`)?

Reply:
I use config.update() to update with received dict, None type would not work IMHO.